src/java/org/apache/cassandra/audit/AuditLogFilter.java

```diff
@@ -126,6 +126,20 @@ private IncludeExcludeHolder(ImmutableSet<String> includedSet, ImmutableSet<Stri
         }
     }
 
+    /**
+     * Checks whether a give AuditLogEntryCategory is filtered or not.
```

Comment:
NIT (spelling): 
```suggestion
     * Checks whether a given AuditLogEntryCategory is filtered or not.
```
Actually applies to the pre-existing comment below as well.